flake8_django/checkers/model_fields.py

```diff
@@ -1,13 +1,18 @@
+import django
 from .checker import Checker
 from .issue import Issue
 
+ALLOW_NULL_BOOLEAN = django.VERSION >= (2, 1)
 
 NOT_NULL_TRUE_FIELDS = [
-    'CharField', 'TextField', 'SlugField',
-    'EmailField', 'UUIDField', 'ImageField',
-    'FileField', 'FilePathField', 'URLField'
+    'CharField', 'TextField', 'SlugField', 'EmailField', 'Field',
+    'UUIDField', 'ImageField', 'FileField'
 ]
-NOT_BLANK_TRUE_FIELDS = ['BooleanField']
+NOT_BLANK_TRUE_FIELDS = []
```

Comment:
The tests are failing now, because coverage decreases from 100% to 99% due to tests on `test_model_fields` where `NOT_BLANK_TRUE_FIELDS` is uses as a parameter. Since the list is empty this tests are not running, so that part of the code is not being covered. 

I think, let's remove `NOT_BLANK_TRUE_FIELDS` altogether, since we could it's ok to have `BooleanField` with `blank=True` even in older version, that's just going to make the system use the default value.

What are your thoughts?